src/wily/commands/rank.py

```diff
@@ -109,19 +109,21 @@ def rank(config, path, metric, revision_index, limit, threshold, descending):
     if limit:
         data = data[:limit]
 
-    # Tack on the total row at the end
-    total = metric.aggregate(rev[1] for rev in data)
-    data.append(["Total", total])
-
-    headers = ("File", metric.description)
-    print(
-        tabulate.tabulate(
-            headers=headers, tabular_data=data, tablefmt=DEFAULT_GRID_STYLE
-        )
-    )
+    if data:
```

Comment:
Can you flip this around to be a [guard-clause](https://blog.boot.dev/clean-code/guard-clauses/#guard-clauses) instead?

```python
if not data:
  return
```

Then it doesn't add another level of nested to the subsequent code.